code/aind_ccf_reg/configs.py

```diff
@@ -0,0 +1,148 @@
+from argschema.fields import Int
+from argschema.fields import List as sch_list
+from argschema.fields import Str 
+
+PathLike = Union[str, Path]
+ArrayLike = Union[dask.array.core.Array, np.ndarray]
+
+VMIN = 0
+VMAX = 1.5
+
+class RegSchema(ArgSchema):
+    """
+    Schema format for Preprocess and Registration.
+    """
+
+    input_data = Str(
+        metadata={
+            "required": True,
+            "description": "Input data without timestamp",
+        }
+    )
+
+    input_channel = Str(
+        metadata={"required": True, "description": "Channel to register"}
+    )
+
+    input_scale = Int(
+        metadata={"required": True, "description": "Zarr scale to start with"}
+    )
+
+    input_orientation = sch_list(
+        cls_or_instance=sch_dict,
+        metadata={
+            "required": True,
+            "description": "Brain orientation during aquisition",
+        },
+    )
+    
+    template_path = Str(
+        metadata={"required": True, "description": "Path to the SPIM template"}
+    ) 
+    
+    ccf_reference_path = Str(
+        metadata={"required": True, "description": "Path to the CCF template"}
+    )
+    
+    template_to_ccf_transform_path =  sch_list(
+        cls_or_instance=Str,
+        metadata={
+            "required": True, 
+            "description": "Path to the transform that aligns SPIM template to CCF"}
+    )
+    
+    ccf_annotation_to_template_moved_path = Str(
+        metadata={"required": True, "description": "Path to the deformed CCF annotation in SPIM template space"}
+    )
+    
+    output_data = Str(
+        metadata={"required": True, "description": "Output file"}
+    )
+    
+    reg_folder = Str(
+        metadata={"required": True, "description": "Folder to save registration results"}
+    )
+    
+    bucket_path = Str(
+        required=True,
+        metadata={"description": "Amazon Bucket or Google Bucket name"},
+    )
+
+    code_url = Str(
+        metadata={"required": True, "description": "CCF registration URL"}
+    )
+
+    metadata_folder = Str(
+        metadata={"required": True, "description": "Metadata folder"}
+    )
+
+    OMEZarr_params = sch_dict(
+        metadata={
+            "required": True,
+            "description": "OMEZarr writing parameters",
+        }
+    )
+    
+    prep_params = sch_dict(
+        metadata={
+            "required": True,
+            "description": "raw data preprocessing parameters",
+        }
+    )
+    
+    ants_params = sch_dict(
+        metadata={
+            "required": True,
+            "description": "ants registering parameters",
+        }
+    )
+
+    reference_res = Int(
+        metadata={
+            "required": True,
+            "description": "Voxel Resolution of reference in microns",
+        }
+    )
+
+    #-------------- DO we need to pass transforms to next capsule ?#
+    # downsampled_file = Str(
+    #     metadata={"required": True, "description": "Downsampled file"}
+    # )
```

Comment:
If in the new version this is not needed anymore, please remove this. I believe these paths were used to know where was the 25 um CCF tiff image (which is no longer needed) and also where to save the transforms. Please, @Di-Wang-AIND make sure to save the forward and reverse transforms for lightsheet to template space:

- Lightsheet to template (two transforms)

The forward and reverse for template to CCF must live in the data asset.